sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+exclude: "^setup/"
+default_language_version:
+  python: python3
+repos:
+- repo: https://github.com/ambv/black
+  rev: 19.3b0
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: trailing-whitespace
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
```

Comment:
We don't really want them, but these files are autogenerated and we don't necessarily control the generator so it's nececssary to ignore them to avoid an infinite fix/regeneration loop.

Reply:
> these files are autogenerated

Then it would be great to have them autogenerated in the pre-commit loop also!